deploy/helm/sumologic/conf/setup/main.tf

```diff
@@ -1,6 +1,6 @@
 terraform {
   required_providers {
-    sumologic  = "~> 2.0.3"
+    sumologic  = "= 2.3.0"
```

Comment:
Should we upgrade to 2.3.1 to get the new deployments?

Reply:
`2.3.1` seems a bit broken because of my fix https://github.com/SumoLogic/terraform-provider-sumologic/pull/92 (it fails on CI)

Let's do `2.3.0`, and when CI passes for next 2.3 release we can upgrade then.